YCAI/src/components/dashboard/lab/RecommendationCard.tsx

```diff
@@ -75,18 +79,37 @@ const useStyles = makeStyles<YCAITheme>((theme) => ({
       marginRight: theme.spacing(1),
     },
   },
+  arrowButton: {
+    '&:disabled': {
+      color: theme.palette.grey[600],
```

Comment:
are you sure we have the `600` for grey?

Reply:
indeed we don't, changed to 500